syntax = "proto2";

package NCloud.NBlockStore.NProto;

option go_package = "github.com/ydb-platform/nbs/cloud/blockstore/config";

////////////////////////////////////////////////////////////////////////////////

message TLocalNVMeConfig {
    // Source URI for retrieving the list of NVMe devices to use (e.g. file:///etc/nbs/devices.txt)
    optional string DevicesSourceUri = 1;

    // Path to the file for persisting the service state.
    // Used to restore state after restart.
    optional string StateCacheFilePath = 2;
}
